### PR TITLE
Move plugin related sub menu items to new Plugins menu item

### PR DESCRIPTION
**Pull request by sgiehl**
### Description:

fixes DEV-17433

Note: Some UI tests are failing because of the menu changes. I've created separate PRs to fix them:

CustomAlerts: https://github.com/matomo-org/plugin-CustomAlerts/pull/153
CustomVariables: https://github.com/matomo-org/plugin-CustomVariables/pull/80
PrivacyManager: https://github.com/matomo-org/matomo/pull/21607
Login (Brutefoce Log): https://github.com/matomo-org/matomo/pull/21608
API error: https://github.com/matomo-org/matomo/pull/21615

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: plugins/CorePluginsAdmin/Menu.php [hunk 1]
```diff
@@ -65,9 +65,8 @@ public function configureAdminMenu(MenuAdmin $menu)
         }
 
         if ($hasSuperUserAccess) {
-            $menu->addSystemItem(Piwik::translate('General_ManagePlugins') . $pluginsUpdateMessage,
-                $this->urlForAction('plugins', array('activated' => '')),
-                $order = 20);
+            $menu->addPluginItem(Piwik::translate('General_ManagePlugins') . $pluginsUpdateMessage,
+                           $this->urlForAction('plugins', array('activated' => '')), 10);
```
Comment (michalkleiner) [inline plugins/CorePluginsAdmin/Menu.php]:
```suggestion
                           $this->urlForAction('plugins', ['activated' => '']), 10);
```

File: plugins/Morpheus/stylesheets/base/icons.css [hunk 2]
```diff
@@ -436,3 +436,12 @@
...
 }
+.icon-plugin:before {
+  content: "\e90d";
+}
+.icon-add1:before {
+  content: "\e62e";
+}
+.icon-remove:before {
+  content: "\e643";
+}
```
Comment (michalkleiner) [inline plugins/Morpheus/stylesheets/base/icons.css:447]: Where are these used? Can't find the usage in this PR.

File: plugins/CorePluginsAdmin/Menu.php (hunk 1 above)
Comment (sgiehl) [reply plugins/CorePluginsAdmin/Menu.php]: This PR is not yet marked as ready for review 🙈

File: plugins/Morpheus/stylesheets/base/icons.css (hunk 2 above)
Comment (sgiehl) [reply plugins/Morpheus/stylesheets/base/icons.css:447]: @bx80 has added those icons to the font in #21077, but they weren't contained in the css file, causing them to be displayed incorrectly on the ui demo, which no one seems to have noticed before. Guess they are meant to be used somewhere at some point 🤷

File: plugins/CorePluginsAdmin/Menu.php (hunk 1 above)
Comment (michalkleiner) [reply plugins/CorePluginsAdmin/Menu.php]: sorry, too quick 🐎

**Review by caddoo — APPROVED** [review]
Tested locally and functionally is good.
Code is fine as well. 

There are some unrelated UI expected screenshots that have been updated, is that just an unrelated clean up so they pass?

**Comment by sgiehl** [comment]
Yes. Those are flaky tests that failed a lot with the previous versions. But they still fail sometimes with the updated files 🤷

**Pull request merged**